### DISCUSSION TITLE
CuyAnimeList versi gw

### DISCUSSION THREAD

**Discussion by Raplici** [deaafrizal/forum#71] [My Projects]
Link: https://www.raplinime.com

untuk alur websitenya tetep sama kyk tutorial, dan beberapa yang gw ubah:

- UI website
- Menambahkan filter pada bagian search keyword untuk mengatasi symbol yang mempengaruhi url seperti "/", dsb.
- Mencoba next-auth versi 5.0.0-beta.4 dan menambahkan: 
a) Login with google
b) Login with credentials (nyoba formik + yup untuk validasi)
c) Validasi email + 2FA untuk login with credentials (nyoba make resend untuk kirim email https://resend.com)
- Menambahkan fitur edit profile untuk merubah nama, email, dan password untuk login with credentials. (untuk login google/github hanya bisa merubah nama)
- Menambahkan fitur edit dan hapus komentar
- Menggunakan query parameter pada popular page, (soalnya kyk kurang nyaman aja semisal udah di page tengah trs ngecek sebuah anime, pas balik malah ke page 1 lagi)

kyknya itu aja wkwk, makasi tutornya bangg

**Comment by rexyb9** [comment]
mantap sih tapi:
- tombol sign in githubnya masih typo

**Reply by Raplici** [reply]
lah iya juga wkwk, makasi udah ngingetin

**Reply by fauzanaslam** [reply]
btw ini hostingnya dmana bg?
